src/io_threads.c

```diff
@@ -227,6 +231,8 @@ static void *IOThreadMain(void *myid) {
     size_t jobs_to_process = 0;
     IOJobQueue *jq = &io_jobs[id];
     while (1) {
+        // Cancellation point so that pthread_cancel() from main thread is honored.
```

Comment:
```suggestion
        /* Cancellation point so that pthread_cancel() from main thread is honored. */
```
Although we allow both, this is the more common style in the codebase.